core/src/main/java/com/netflix/conductor/model/TaskModel.java

```diff
@@ -148,11 +151,15 @@ public boolean isRetriable() {
     private String subWorkflowId;
 
     /**
-     * Use to note that a sub workflow associated with SUB_WORKFLOW task has an action performed on
+     * Used to note that a sub workflow associated with SUB_WORKFLOW task has an action performed on
      * it directly.
      */
     private boolean subworkflowChanged;
 
+    @JsonIgnore private Map<String, Object> inputPayload = new HashMap<>();
```

Comment:
Some comment here to help reader to distinguish between `inputData` vs `inputpayload`, `outputData` vs `outputpayload` and why those are Json ignored. Also what do you think of rename this to be more explicitly indicate its for the externalized storage? e.g. `inputExternalized`?

Reply:
The model classes are serialized to Json, if the fields are ignored, they will be stored in the database. In case of large input or output, only the reference (*storagePath fields) needs to be stored.

Reply:
ya got it. I meant to add a comment in the code for what is the field for for reference to easily understand the code in future